lib/CL/devices/common_utils.h

```diff
@@ -55,6 +55,7 @@ struct kernel_run_command
   kernel_run_command *prev;
   kernel_run_command *next;
   unsigned long ref_count;
+  size_t unreachable;
```

Comment:
Perhaps comment what this member means/is for. By the looks of the other changes, it seems to count the times WIs reached unreachable parts of code - is that correct?

Reply:
renamed the flag & added a comment